### PR TITLE
Support Version 2.0.0 changes for /v1/bequeath-your-data-and-die #104

### PR DESCRIPTION
**Pull request by sunil-kumar-meena**
fixed below isses

https://github.com/openBackhaul/TypeApprovalRegister/issues/102
 https://github.com/openBackhaul/TypeApprovalRegister/issues/103

Note: @vanithavalluripalli9 is the reviewer

### CODE REVIEW

File: server/service/IndividualServicesService.js [hunk 1]
```diff
@@ -59,43 +62,58 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
...
-        let currentApplicationRemoteAddress = await TcpServerInterface.getLocalAddress();
-        let currentApplicationRemotePort = await TcpServerInterface.getLocalPort();
-        if((applicationAddress == currentApplicationRemoteAddress) && 
-        (applicationPort == currentApplicationRemotePort)){
-          isdataTransferRequired = false;
+      let logicalTerminationPointConfigurationStatus = {};
+      let newReleaseHttpClientLtpUuid = HttpClientLtpUuidFromForwarding[0];
+      if (newReleaseHttpClientLtpUuid != undefined) {
+      
+        let isReleaseUpdated = await httpClientInterface.setReleaseNumberAsync(newReleaseHttpClientLtpUuid, releaseNumber);
```
Comment (vanithavalluripalli9) [inline server/service/IndividualServicesService.js:85]: As set operation will always return true if write to the database is successful and it will not check for the value update . We have to include a check before set operation if incoming value is different from the value present if so then set the value. In this way we can assure wrong forwarding is not sent to ALT. Please include this change for all set attribute methods.

File: server/service/IndividualServicesService.js [hunk 2]
```diff
@@ -360,27 +409,36 @@ exports.redirectInfoAboutApprovalStatusChanges = function (body, user, originato
...
       /****************************************************************************************
        * Prepare logicalTerminatinPointConfigurationInput object to 
        * configure logical-termination-point
        ****************************************************************************************/
 
-      let operationList = [
-        subscriberOperation
+      let operationNamesByAttributes = new Map();
+      operationNamesByAttributes.set("subscriber-operation", subscriberOperation);
+
```
Comment (vanithavalluripalli9) [inline server/service/IndividualServicesService.js:421]: These changes are not part of this pull request . Please remove them

File: server/service/individualServices/IndividualServicesOperationsMapping.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module.exports.individualServicesOperationsMapping =
+{
+  "/v1/redirect-info-about-approval-status-changes": {
+    "subscriber-operation": {
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/IndividualServicesOperationsMapping.js:4]: These changes are not part of this pull request.please remove them.

File: server/service/individualServices/SoftwareUpgrade.js [hunk 4]
```diff
@@ -217,7 +220,7 @@ async function PromptForBequeathingDataCausesTransferOfListOfAlreadyGrantedTypeA
                      ************************************************************************************/
                     let requestBody = {};
                     requestBody.applicationName = applicationName;
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/SoftwareUpgrade.js:222]: could you please fetch the information from application-data.json file instead of application profile as application profile is no more valid from v2.0.0

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

File: server/service/IndividualServicesService.js (hunk 1 above)
Comment (sunil-kumar-meena) [reply server/service/IndividualServicesService.js]: done

File: server/service/IndividualServicesService.js (hunk 2 above)
Comment (sunil-kumar-meena) [reply server/service/IndividualServicesService.js]: done

File: server/service/individualServices/IndividualServicesOperationsMapping.js (hunk 3 above)
Comment (sunil-kumar-meena) [reply server/service/individualServices/IndividualServicesOperationsMapping.js]: done

File: server/service/individualServices/SoftwareUpgrade.js (hunk 4 above)
Comment (sunil-kumar-meena) [reply server/service/individualServices/SoftwareUpgrade.js:225]: done

**Review by vanithavalluripalli9 — APPROVED** [review]
